app/hotels/src/allHotels/searchForm/DateInput.js

```diff
@@ -15,7 +15,7 @@ type Props = {|
 
 type DateInputType = 'checkin' | 'checkout';
 
-export const DISPLAY_DATE_FORMAT = 'DD/MM/YYYY';
+export const DISPLAY_DATE_FORMAT = 'MMM DD';
```

Comment:
If we are going to use this kind of date formatting, we also need to set 
```
import moment from 'moment/min/moment-with-locales`
const deviceLocale = //getfrom native code
moment.locale(deviceLocale);
```

 Since on my device with norwegian, it shows `Dec 18`,  while expected output would be `des. 18`;
Which probably just means that we need to do #246

Reply:
Seems like this is more of a `react-native-datepicker` issue

https://github.com/xgfe/react-native-datepicker/issues/71

Reply:
I would prefer not to fix it now since we didn't do localization at all yet. What do you think?